pytest/tests/loadtest/locust/common/base.py

```diff
@@ -259,6 +289,14 @@ def poll_tx_result(self, meta, params):
     def account_exists(self, account_id: str) -> bool:
         return "error" not in self.node.get_account(account_id, do_assert=False)
 
+    def create_contract_account(self,
+                                funding_account: Account,
+                                key: key.Key,
+                                balance: float = 50000.0):
```

Comment:
Maybe we can use `int` instead of `float` here?

Reply:
yeah makes sense